rpi/demos/README.md

```diff
@@ -0,0 +1,41 @@
+# WALKTHROUGH 
+
+### For these exampls we will be using Python
+
+Let us begin a walkthrough with your Pi and a brief introduction into various aspects.
+First please be aware:
+New command for raspberry pi which allow to view the GPIO pin layouts:  Open a terminal and type `pinout`This will provide a layout within the terminal of the current raspberry pi pins. 
```

Comment:
```suggestion
New command for raspberry pi which allow to view the GPIO pin layouts:  Open a terminal and type `pinout`

This will provide a layout within the terminal of the current raspberry pi pins. 
```